codecov.yml

```
@@ -2,7 +2,7 @@ fixes:
   - "pennylane_qchem/::qchem/pennylane_qchem/"
 
 ignore:
-  - "pennylane/plugins/tests/*"
+  - "pennylane//tests/*"
```

```suggestion
  - "pennylane/tests/*"
```

---

wait, shouldn't this be
```suggestion
  - "pennylane/devices/tests/*"
```

---

Not sure what happened there!